src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -86,6 +86,8 @@ internal static class IDEDiagnosticIds
 
         public const string FormattingDiagnosticId = "IDE0054";
 
+        public const string MoveMisplacedUsingsDiagnosticId = "IDE0055";
```

Comment:
will conflict with other PRs going in.

Reply:
Idea: If blank lines between these were removed, it would show up as an actual conflict so that people would be alerted of this and fix it.

Reply:
Actually it should be a conflict anyway, forget what I'm saying.

Reply:
Is there an easy way to ask GitHub for all the open PRs that modify this file? =)